Generics.md

```diff
@@ -0,0 +1,103 @@
+# Generic Documents
+
+Generic documents are documents which conform to the "Generic" API in CMR. These are JSON documents validated against a known [schema][schema] file.
+
+
+## Configuration
+
+If adding a new document, you will need to update the defconf variable by either setting an ENV for global change, or by updating the default value in [/common-lib/src/cmr/common/config.clj](/common-lib/src/cmr/common/config.clj). The format for this value is either JSON for an ENV variable or a clojure map if setting directly in the default attribute of the defconfig like this:
+
+	(defconfig approved-pipeline-documents
+  		{:default {:grid ["0.0.1"]
+             :dataqualitysummary ["1.0.0"]
+             :orderoption ["1.0.0"]
+             :serviceentry ["1.0.0"]
+             :serviceoption ["1.0.0"]}
+   		:parser #(json/parse-string % true)})
```

Comment:
Will this render as a code block? I thought it needed 4 spaces if it's not using the tick mark notiation

Reply:
if you mean does it work, yes it does.